ecosystem/plugin-search/src/client/composables/useHotKeys.ts

```diff
@@ -18,7 +18,8 @@ export const useHotKeys = ({
       // key matches
       isKeyMatched(event, hotKeys.value) &&
       // event does not come from search box
-      !input.value.contains(event.target as Node)
+      !input.value.contains(event.target as Node) &&
```

Comment:
Is the condition redundant after adding `isFocusingTextControl` ?

Reply:
Yes, it is. The search box (rather what's referenced here as `input.value`) is an input element so it's also covered by `isFocusingTextControl`. I've removed the superfluous condition and left a comment.